package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "atdgen-workshop-starter",
+  "version": "0.1.0",
+  "scripts": {
+    "build": "bsb -make-world",
+    "start": "bsb -make-world -w -ws _ ",
```

Comment:
What is this `-ws` parameter?

Reply:
It's an (undocumented) flag to tell bsb to use the websocket connection to detect changes I think: https://github.com/BuckleScript/bucklescript/blob/1ba584d3d5e0a5e96adb42f0cc7c2f39427de5ed/lib/bsb#L110